test/unit/suite.spec.js

```diff
@@ -680,6 +680,29 @@ describe('Suite', function() {
       });
     });
   });
+
+  describe('.markOnly()', function() {
+    beforeEach(function() {
+      sandbox = sinon.createSandbox();
```

Comment:
This beforeEach and the `afterEach` beneath it should not be there. The suite already creates a `sandbox`, you're creating another one here and assigns it to the same variable. This means the `sandbox` that the suite creates will become unreachable and any stubs that were registered on it would never be `restore`d.

Reply:
Ah no sorry my bad. I removed them.